### ISSUE TITLE
path, README, gxrun

### ISSUE THREAD

**Issue by markito3** [JeffersonLab/hd_singularity#5]
In the README in scripts/gxrun, for the examples to work, gxrun has to be in the path. That requirement is not mentioned, no?

**Comment by s6pepaul** [comment]
you are right, I will make some changes to the descriptions.... Until then I leave this open as reminder to myself.
